addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -112,6 +112,7 @@ import wUtils from '@website/js/utils';
                 const defaultValue = input.getAttribute("value");
                 this.call("datetime_picker", "create", {
                     target: input,
+                    onChange: this._onFieldInputDebounced,
```

Comment:
Wouldn't it be more accurate to trigger the native input event (eg something like `.dispatchEvent(new Event('input', { bubbles: true }));`) which would then make the field behaves as others (being listened on the input event and entering `_onFieldInputDebounced` since it's listened at line 107).

Reply:
Indeed, it seems a more stable diff to me since it preserves other "input" handlers, if any.
Code updated :+1: